domain/storage/state/storagepool.go

```diff
@@ -454,29 +454,49 @@ WHERE  sp.type IN ($storageProviderTypes[:])`, spTypes, storagePool{}, poolAttri
 	return dbRows.toStoragePools(keyValues)
 }
 
-// GetStoragePoolByName returns the storage pool with the specified name.
+// GetStoragePoolUUID returns the UUID of the storage pool with the specified name.
 // The following errors can be expected:
 // - [storageerrors.PoolNotFoundError] if a pool with the specified name does not exist.
-func (st State) GetStoragePoolByName(ctx context.Context, name string) (domainstorage.StoragePool, error) {
+func (st State) GetStoragePoolUUID(ctx context.Context, name string) (domainstorage.StoragePoolUUID, error) {
 	db, err := st.DB()
 	if err != nil {
-		return domainstorage.StoragePool{}, errors.Capture(err)
+		return "", errors.Capture(err)
 	}
-	return GetStoragePoolByName(ctx, db, name)
+	return getStoragePoolUUID(ctx, db, name)
 }
 
-// GetStoragePoolByName returns the storage pool with the specified name.
+func getStoragePoolUUID(
```

Comment:
What is the purpose of this version? Is this going to be implemented in this PR?